pkgs/leak_tracker_flutter_test/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: leak_tracker_flutter_test
+version: 0.0.1
+description: Tests for leak_tracker that depend on Flutter framework.
+repository: https://github.com/dart-lang/leak_tracker
+
+environment:
+  sdk: '>=3.0.0 <4.0.0'
+
+dev_dependencies:
+  flutter:
+    sdk: flutter
+  flutter_lints: ^2.0.0
+  flutter_test:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
+  leak_tracker: ^7.0.4
```

Comment:
Do you want to use a `path: ../leak_tracker` reference here? Or `leak_tracker: any` and then have a `dependency_overrides:` entry with `path: ../leak_tracker`; either will let you run against the repo version of leak_tracker.

Reply:
```
Publishable packages can't have 'path' dependencies.
Try adding a 'publish_to: none' entry to mark the package as not for publishing or remove the path dependency.
```